src/main/scala/edu/ie3/simona/model/SystemComponent.scala

```diff
@@ -40,7 +40,7 @@ abstract class SystemComponent(
 
   private val elementType: String = this.getClass.getSimpleName
 
-  // check if a uuid is provided
+  // check if an uuid is provided
```

Comment:
I think `a` is correct here, since _pronounciation_ counts, and UUID sounds like "you-you-i-d". [Wikipedia agrees.](https://en.wikipedia.org/wiki/Universally_unique_identifier)